smoke-tests/src/test/groovy/io/opentelemetry/smoketest/QuarkusSmokeTest.groovy

```diff
@@ -22,6 +23,11 @@ class QuarkusSmokeTest extends SmokeTest {
     "ghcr.io/open-telemetry/opentelemetry-java-instrumentation/smoke-test-quarkus:jdk$jdk-20211213.1574595137"
   }
 
+  @Override
+  protected TargetWaitStrategy getWaitStrategy() {
+    return new TargetWaitStrategy.Log(Duration.ofMinutes(1), ".*Listening on.*")
+  }
```

Comment:
thx for adding all of these 👍

Reply:
Just want to confirm if it's anything to be concerned about (file an issue about)? I've never seen startup time failures until this PR I think

Reply:
let's see what @laurit thinks, he's been looking at the smoke tests recently (e.g. #5772)

Reply:
I think this is the correct thing to do. It is hard to explain why these are suddenly needed but for example `SpringBootSmokeTest` already had it while it was added to `SpringBootWithSamplingSmokeTest` in this pr. So might as well argue that it was strange that `SpringBootWithSamplingSmokeTest` worked before.